core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -572,15 +574,15 @@ public void putSplits(String tableName, SortedMap<Text,TabletMergeability> split
             throw new CompletionException(e);
           }
           // wait for the fate operation to complete in a separate thread pool
-        }, startExecutor).thenApplyAsync(pair -> {
-          final TFateId opid = pair.getFirst();
-          final List<Pair<Text,TabletMergeability>> completedSplits = pair.getSecond();
```

Comment:
I suspect these variables were created to make the code more readable and avoid getFirst and getSecond.  After replacing w/ a record I inlined the variables.